tmt-app/src/Components/InfoLinkPage.jsx

```diff
@@ -26,41 +28,72 @@ function MajorPage() {
       name: "BobaTalks Discord Server",
       link: "https://discord.com/invite/bobatalks",
     },
+    ...(params.career
+      ? [
+          {
+            name: "Career Information via Zippia",
+            link: data.link,
+          },
+        ]
+      : []),
   ];
-  let sectionInfo = [
-    {
-      title: `What is ${
-        ["a", "e", "i", "o", "u"].includes(data.name[0].toLowerCase())
-          ? "an "
-          : "a "
-      }
+  let sectionInfo = params.major
+    ? [
+        {
+          title: `What is ${
+            ["a", "e", "i", "o", "u"].includes(data.name[0].toLowerCase())
```

Comment:
If we are deciding to combine these, why not just use a map and access the list from a key?

list[params.infoType] 

Since we were making a change to the url and routing anyways. We can add in :type and :id or something for example

Reply:
Would it be what we have right now because we are using :major or :career as a key to access our list.
![image](https://github.com/BobaTalks/pick-a-major/assets/36683390/81a65e45-8367-44a1-890a-76bc3d72789e)